.pre-commit-config.yaml

```diff
@@ -0,0 +1,31 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.5.0
+  hooks:
+    - id: trailing-whitespace
+    - id: end-of-file-fixer
+    - id: check-yaml
+    - id: check-added-large-files
+
+- repo: https://github.com/ansible-community/ansible-lint
+  rev: v24.2.1
+  hooks:
+    - id: ansible-lint
+      always_run: false
+      files: ^.*.yml
+      exclude: .github
+      additional_dependencies:
+        - ansible-core>=2.16.0
```

Comment:
See comment about ansible-core version

Reply:
Trying to find a version for ansible-lint which will make the existing roles pass a check is more difficult than I expected. I suggest just accepting that the roles will not pass lint now, letting the pipeline fail, and slowly converge in the Big PR.